modules/openxr/extensions/openxr_composition_layer_extension.h

```diff
@@ -78,20 +103,37 @@ class OpenXRViewportCompositionLayerProvider {
 	Dictionary extension_property_values;
 	bool extension_property_values_changed = true;
 
-	RID viewport;
-	Size2i viewport_size;
-
-	OpenXRAPI::OpenXRSwapChainInfo swapchain_info;
+	struct SubViewport {
```

Comment:
Should we use `ViewportInfo` to avoid confusion with the `SubViewport` node?

Reply:
Hm, yeah, I could see that being potentially confusing.

We actually don't even need to give this struct a name, since there's only ever the one instance of it. The same goes for the `AndroidSurface` struct that comes after. I'll just remove both names.